themes/src/main/resources/theme/keycloak-preview/account/resources/content.json

```diff
@@ -56,5 +56,14 @@
     "modulePath": "/content/my-resources-page/MyResourcesPage.js",
     "componentName": "MyResourcesPage",
     "hidden": "!features.isMyResourcesEnabled"
+  },
+  {
+    "id": "deleteAccount",
+    "path": "deleteAccount",
```

Comment:
The path will be part of the URL to this page.  So "delete-account" is probably better than "deleteAccount".

Reply:
done!